src/test/java/com/dsrm/dsrmbackend/admin/AdminRoomControllerTests.java

```diff
@@ -180,4 +183,22 @@ public void addRoomThroughUpdate() throws Exception {
         assertEquals(LocalTime.parse("12:00:00"), room.getOpeningTime());
         assertEquals(LocalTime.parse("22:00:00"), room.getClosingTime());
     }
+
+    @Test
+    @Transactional
+    public void deleteExistentRoom() throws Exception {
+        this.mockMvc.perform(delete("/admin/rooms/1")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNoContent());
+        assertThat(reservationRepo.findById("1")).isEmpty();
+    }
+
+    @Test
+    @Transactional
+    public void deleteNonExistentRoom() throws Exception {
+        this.mockMvc.perform(delete("/admin/rooms/4123")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNoContent());
+        assertThat(roomRepo.findById("4123")).isEmpty();
+    }
```

Comment:
I think this test is unnecessary because such a record doesn't exist anyway so checking if it doesn't exist misses the point